.github/pull_request_template.md

```diff
@@ -1,69 +1,78 @@
+- [ ] [🛠][1] New dependencies are added to the [project requirements](https://docs.esmvaltool.org/en/latest/community/diagnostic.html#additional-dependencies)
+- [ ] [🧪][2] [Documentation](https://docs.esmvaltool.org/en/latest/community/diagnostic.html#documentation) for the recipe/diagnostic clearly describes what it calculates from a scientific point of view
+- [ ] [🧪][2] Recipe runs successfully on [`@esmvalbot`](https://docs.esmvaltool.org/en/latest/community/introduction.html#running-tests) or some other machine without modification
+- [ ] [🧪][2] Figure(s)/data [look as expected](https://docs.esmvaltool.org/en/latest/community/review.html#scientific-review)
+- [ ] [🧪][2] Code is [well documented](https://docs.esmvaltool.org/en/latest/community/introduction.html#what-should-be-documented) and scientifically sound
 
-New or updated [recipe/diagnostic](https://docs.esmvaltool.org/en/latest/community/diagnostic.html):
+### New or updated [data reformatting script](https://docs.esmvaltool.org/en/latest/develop/dataset.html):
 
-- [ ] [The documentation](https://docs.esmvaltool.org/en/latest/recipes/index.html) for the new/updated recipes/diagnostics clearly describes what the recipe does and how to use it
-- [ ] The recipe runs successfully on your own machine/cluster or with the [`@esmvalbot`](https://github.com/apps/esmvalbot) without any modifications to the recipe and with all data specified in the recipe
-- [ ] The figure(s) and data look as expected from the literature and/or the paper that is reproduced by the recipe
-- [ ] The code contains comments with references to formulas, figures, tables, etc. that are used from papers/online resources
+- [ ] [🛠][1] Dataset is added to the [table in the documentation](https://docs.esmvaltool.org/en/latest/community/dataset.html#dataset-documentation)
+- [ ] [🛠][1] Documentation contains [instructions to obtain the data](https://docs.esmvaltool.org/en/latest/community/dataset.html#dataset-documentation)
+- [ ] [🛠][1] [Tests for the CMORized data](https://docs.esmvaltool.org/en/latest/community/dataset.html#dataset-tests) are available
+- [ ] [🧪][2] Numbers/units of the data look [physically meaningful](https://docs.esmvaltool.org/en/latest/community/dataset.html#cmorizer-output)
+- [ ] [🛠][1] Data set is added to the [OBS data pool](https://docs.esmvaltool.org/en/latest/community/dataset.html#adding-your-dataset-to-the-obs-data-pool)
 
-New or updated [data reformatting script](https://docs.esmvaltool.org/en/latest/develop/dataset.html):
+***
 
-- [ ] The numbers and units of the data look physically meaningful
+To help with the number pull requests:
 
+- [ ] 🙏 We kindly ask you to [review](https://docs.esmvaltool.org/en/latest/community/review.html#review-of-pull-requests) two other [open pull requests](https://github.com/ESMValGroup/ESMValTool/pulls) in this repository
```

Comment:
I would leave the checkbox here, this might make the impression that the pull request cannot be finished before another 2 pull request have been reviewed....this is not what we want correct? Otherwise I like it, its good that people know how this is supposed to work. But if it is your very first pull request it might scare you off if it looks like a "have to".

Reply:
So you suggest to remove the checkbox, but leave the item?

Reply:
Yes, agree we should not scare of first time contributors. :-)

Reply:
I removed the checkbox in the latest commit, so now it is more like a plea to get people to help out 👍